### PR TITLE
Using classes to control styles instead of HTML tags.

### PR DESCRIPTION
**Pull request by tomdaniels**

### CODE REVIEW

**Comment by rayvafa** [comment]
There is few more styles in the `css` file that don't have class names(still using tag names).

Also it would be nice if you could use more semantic names for your css classes. have a read in the following article, it's really good one: maybe use more semantic names for the class names: https://css-tricks.com/semantic-class-names/

**Comment by rayvafa** [comment]
also when you use classes, you can remove the tag from your css. for example instead of having:
```
h1.head1 {
  color: white;
}
h2.head2 {
  color: red;
}
```
you can go:
```
.main-headline {
  color: white;
}
.sub-headline {
  color: red;
}
```

**Comment by tomdaniels** [comment]
@rayvafa have re-pushed updated files. Not sure I fully understood the semantic tags from the article. There are a few that are still not semantic just wanted to see if I was on the right track here first?

File: index.html [hunk 1]
```diff
@@ -12,9 +12,13 @@
 </head>
 
 <body>
-  <h1>Bate Bay Boards</h1>
-  <h2> Coming Soon </h2>
-  <h3> Handmade | Hand-Shaped </h3>
+  <main-headline>
```
Comment (rayvafa) [inline index.html]: we don't have custom tags. at least not at the moment. so the tag will look like:
```
<h1 class="main-headline">Bate Bay Boards</h1>
```

and css would be like:
```
.main-headline {
   text-align: right;
}
```

so you css class names need to be semantic. that tag name will stay the same.

Comment (tomdaniels) [inline index.html]: Ok, think I've got. 

Have just pushed new changes.

**Comment by tomdaniels** [comment]
@rayvafa Just wondering if you saw the last commit made?

File: styles.css [hunk 2]
```diff
@@ -1,21 +1,13 @@
...
   text-align: right;
   color: white;
 }
-h3 {
-  display: block;
-  margin: auto;
-  text-align: right;
-  color: #ffffff;
-}
 img.centre {
```
Comment (rayvafa) [inline styles.css]: nice, looks much better. there is still few more like this `img` tag in the `css` file. please have a look and make sure there is no mentioning of `tag names` in `css` file. just `css` classes allowed there.

Comment (tomdaniels) [inline styles.css]: @rayvafa the img tag is not the html tag, it sets the background image. I haven't used the image in index.html? so it doesn't have a class assigned to it.

Comment (rayvafa) [inline styles.css]: oh, yeah. there is no `img` tag on the `html` file any more. so I think you can remove `img.center` completely. it doesn't have anything to do with background image.

File: styles.css [hunk 3]
```diff
@@ -32,20 +38,20 @@ a.btn:hover {
   background: white;
   color: seagreen;
 }
-a {
+a.btn {
```
Comment (rayvafa) [inline styles.css]: instead of `a.btn` we can go just `.btn`. it does the same thing but it will remove the tag name from css. which is nicer.

**Comment by tomdaniels** [comment]
@rayvafa All html tags have been removed from stylesheet. 

Has been committed and pushed.

**Pull request merged**